src/library/parsercsv.cpp

```diff
@@ -21,10 +7,26 @@
 
 #include "moc_parsercsv.cpp"
 
-ParserCsv::ParserCsv() : Parser() {
+namespace {
+
+bool isColumnExported(BaseSqlTableModel* pPlaylistTableModel, int column) {
+    if (pPlaylistTableModel->isColumnInternal(column)) {
+        return false;
+    }
+    if (pPlaylistTableModel->fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_PREVIEW) == column) {
+        return false;
+    }
+    if (pPlaylistTableModel->fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COVERART) == column) {
+        // This is the bas64 encoded image which may hit the maximum line length of spreadsheet applications
+        return false;
+    }
+    return true;
 }
 
-ParserCsv::~ParserCsv() {
+} // namespace
+
+ParserCsv::ParserCsv()
```

Comment:
Please replace this redundant definition with a `= default;` declaration (or no declaration at all if it is implicitly generated).